devcycle_python_sdk/managers/config_manager.py

```diff
@@ -82,7 +82,7 @@ def _get_config(self, last_modified: Optional[float] = None):
             json_config = json.dumps(self._config)
             self._local_bucketing.store_config(json_config)
             if not self._options.disable_realtime_updates:
-                if self._sse_manager is None:
+                if self._sse_manager is None or not self._sse_manager.client.is_connected():
```

Comment:
does this mean you create a new SSEManager when the client isn't connected? wouldn't it be better to do some sort of reconnection strategy?

Reply:
yea if this becomes an issue i think we should revisit this. gonna merge this for now --> Its a pre-emptive solution to an anticipated problem.